### ISSUE TITLE
Final output

### ISSUE THREAD

**Issue by NarendranM08** [gan-police/frequency-forensics#24]
Can you share the screenshot of  final output  where the images are labeled as fake or real

**Comment by v0lta** [comment]
Are you asking for a front-end? The code in this repository allows training and executing a backend. But is mainly intended to back up the results we report in the paper. We do not provide a frontend here, but it could of course be built using Django for example.

**Comment by NarendranM08** [comment]
No , like labelling from the dataset that the images are real or fake

**Comment by v0lta** [comment]
The label is assigned based on the file path in https://github.com/gan-police/frequency-forensics/blob/c7e12968b14e0f609ab7e5d6df7d4090c85819f3/src/freqdect/prepare_dataset.py#L80 .
For this approach to work, we require a specific file path. I.e for example:
source_data
 ├── A_original
 ├── B_CramerGAN
 ├── C_MMDGAN
 ├── D_ProGAN
 └── E_SNGAN

**Comment by NarendranM08** [comment]
Should we label the folder by ourselves or running the project code

**Comment by v0lta** [comment]
This has to happen by hand. For the code to work the folders must be arranged properly.

**Comment by NarendranM08** [comment]
So from the drive that A_ffhq, b_Stylegan we make into five directories
 source_data
├── A_original
├── B_CramerGAN
├── C_MMDGAN
├── D_ProGAN
└── E_SNGAN

**Comment by v0lta** [comment]
No, FFHQ means Flickr Face High Quality and StyleGAN is a GAN, see section 5.1 of the paper for more information. That means the downloaded archive has images from two sources and works as-is. The file structure with five sources (A,B,C,D,E) is what we used in section 5.2 in the paper.

**Comment by NarendranM08** [comment]
So based  on folder labels it presents that the image is real or fake

**Comment by v0lta** [comment]
Only the training pipeline is set up using folder labels. Once a classifier is trained the folder structure is no longer relevant.
Feel free to play with the pre-trained network from https://github.com/gan-police/frequency-forensics/issues/22 .

**Comment by NarendranM08** [comment]
Is the wavelet-packet creation done in  fourier_math.py
